htdocs/src/Oc/Util/DbalConnection.php

```diff
@@ -37,7 +53,7 @@ public static function createDbalConnection(
             $params['password'] = $password;
         }
 
-        if ($port) {
+        if ($port !== null) {
```

Comment:
why? I think the if condition is the same?

Reply:
port number can be an integer '0' and would be evaluated as false.
Scrutinizer told this :tongue:

Reply:
but port 0 doesn't work :-*